src/ngx_http_lua_timer.c

```diff
@@ -134,6 +155,10 @@ ngx_http_lua_ngx_timer_at(lua_State *L)
 
     delay = (ngx_msec_t) (luaL_checknumber(L, 1) * 1000);
 
+    if (every && delay == 0) {
+        return luaL_error(L, "delay cann't be zero");
```

Comment:
Type: "cann't". Should use "can't" instead. But I think "cannot" is better for error messages.